packages/test-ui/src/components/Swap.tsx

```diff
@@ -238,17 +241,20 @@ export const Swap = ({ web3Provider, signer, chainData }: SwapProps): ReactEleme
     init();
   }, [web3Provider, signer]);
 
-  const getUserBalance = async (chainId: number, _signer: Signer) => {
+  const getUserBalance = async (_chainId: number, _signer: Signer) => {
+    if (_chainId === 0) {
+      return BigNumber.from(0);
```

Comment:
im not allowed to have monies on mainnet ? :(

Reply:
mainnet id is 1, u r not allowed to have monies on 0 which is not a chain

Reply:
no fake monies 4 u